src/modbus_proxy.py

```diff
@@ -171,13 +172,30 @@ async def _write_read(self, data):
         await self._write(data)
         return await self._read()
 
+    def _transform_request(self, request):
+        request_unit_id = request[6]
+
+        if self.unit_id_remapping is None:
+            return request
+
+        if self.unit_id_remapping[request_unit_id] is None:
+            return request
+
+        remapped_unit_id = self.unit_id_remapping[request_unit_id]
+
+        self.log.info("remapping unit ID %s to %s", request_unit_id, remapped_unit_id)
```

Comment:
I'd like to propose `debug` log level for this kind of messages

Reply:
Done.